### PR TITLE
Bugfix/issues 50 52

### LINKED ISSUE #50: improvement: after bounty is created, automatically refresh list
Currently, once a bounty is created, a user will need to click "Back to List" and then "refresh" to see the created bounty on the main list of bounties. The second action with hitting "refresh" will be confusing for users - they aren't aware of the technical challenge we have with doing this, and it's an un-intuitive UI

After a bounty is created, I'd like the list to be automatically refreshed so that the user doesn't have to hit "refresh"

### PR DESCRIPTION
**Pull request by thedarkrai07**
Resolves #50 #51 #52

### CODE REVIEW

**Review by tescher — CHANGES_REQUESTED** [review]
Great work!

I think the intention for the buttons instead of links was to also include them on activityResponse in addition to activityNotification.

While you're at it, the Help activity probably needs to use activityResponse instead of .send so we can centralize error handling when we do that.

**Comment by thedarkrai07** [comment]
Have updated and pushed the changes. have not extensively tested it yet

**Review by tescher — CHANGES_REQUESTED** [review]
Still having the link in the content doesn't really make sense, it's redundant with the button. Let's get rid of the link, and maybe reword the message to include the bounty title so the user knows what bounty is being referenced without having to jump to it. What do you think?
image

**Review by tescher — CHANGES_REQUESTED** [review]
Getting an error when applying for a "require-application" bounty. This seems to happen when you are calling activityResponse with no bounty link, and it is in response to a button press. It calls interactionResponse, which creates an empty MessageActionRow, which fails. It also fails in Delete and probably elsewhere.

```
In Apply activity
62f3b519e9f037acd980a86f bounty applied for by chkaloon#6313
error
{
  "indexMeta": true,
  "meta": {
    "name": "Error",
    "message": "Invalid Form Body\ndata.components[0].components: This field is required",
    "stack": "Error: Invalid Form Body\ndata.components[0].components: This field is required\n    at Object.interactionResponse (/home/tescher/bountybot-/dist/app/utils/DiscordUtils.js:144:23)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async Object.activityResponse (/home/tescher/bountybot-/dist/app/utils/DiscordUtils.js:162:13)\n    at async applyBounty (/home/tescher/bountybot-/dist/app/activity/bounty/Apply.js:30:5)\n    at async Object.run (/home/tescher/bountybot-/dist/app/activity/bounty/ActivityHandler.js:47:17)\n    at async handler (/home/tescher/bountybot-/dist/app/activity/bounty/Handler.js:32:5)\n    at async default_1.interactionHandler (/home/tescher/bountybot-/dist/app/events/InteractionCreate.js:231:13)\n    at async default_1.execute (/home/tescher/bountybot-/dist/app/events/InteractionCreate.js:62:9)"
  }
}
/home/tescher/bountybot-/dist/app/utils/DiscordUtils.js:144
                throw new RuntimeError_1.default(e);
                      ^

RuntimeError: Invalid Form Body
data.components[0].components: This field is required
    at Object.interactionResponse (/home/tescher/bountybot-/dist/app/utils/DiscordUtils.js:144:23)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async default_1.interactionHandler (/home/tescher/bountybot-/dist/app/events/InteractionCreate.js:255:20)
    at async default_1.execute (/home/tescher/bountybot-/dist/app/events/InteractionCreate.js:62:9)
[nodemon] app crashed - waiting for file changes before starting...
```

**Review by Behold3th — APPROVED** [review]
🔥 work @thedarkrai07 and @tescher

**Review by tescher — APPROVED** [review]

**Comment by Behold3th** [comment]
👏👏
